src/gui/transferlistwidget.cpp

```diff
@@ -552,7 +552,7 @@ void TransferListWidget::openSelectedTorrentsFolder() const
     // folders prehilighted for opening, so we use a custom method.
     for (BitTorrent::Torrent *const torrent : asConst(getSelectedTorrents()))
     {
-        const Path contentPath = torrent->actualStorageLocation() / torrent->contentPath();
+        const Path contentPath = torrent->contentPath();
         paths.insert(contentPath);
     }
     MacUtils::openFiles(PathList(paths.cbegin(), paths.cend()));
```

Comment:
Off topic (and not a blocker for this PR), there is another minor performance issue here: it would be better to change to `MacUtils::openFiles(const QSet<Path> &paths)` so that we don't need to (redundantly) convert to list type.

Reply:
I understand, creating another container (even when copying elements is cheap) is pretty expensive operation, but in this particular place it doesn't matter.
also, as for for me, `PathList` looks good there from "philosophy" point of view. `QSet` is "implementation detail" in particular case, it doesn't worth changing function signature depending on that particular use case.

Reply:
It is ok that it is left out of this PR.

The below is just some chit chat.
>but in this particular place it doesn't matter

From my experience, every places matter (sooner or later).

>QSet is "implementation detail" in particular case

IMO every concrete type is an implementation detail (including `PathList`). Only generics or traits that implements "iterable" concepts would be free (or less) of those "implementation details".

Reply:
> Only generics or traits that implements "iterable" concepts would be free (or less) of those "implementation details".

Unfortunately, the interface-oriented approach is not very developed in the C++ ecosystem (at least in Qt, and in the standard library too). Instead, generalized template-based programming is often proposed, which leads to the generation of additional binary code.